src/Config/Config.ts

```diff
@@ -448,7 +448,9 @@ export class BridgeConfig {
     @configKey("Basic homeserver configuration")
     public readonly bridge: BridgeConfigBridge;
     @configKey(`Configuration for encryption support in the bridge.
- If omitted, encryption support will be disabled.`, true)
+ If omitted, encryption support will be disabled.
+ This feature is HIGHLY EXPERIMENTAL AND SUBJECT TO CHANGE.
+ For more details, see https://github.com/matrix-org/matrix-hookshot/issues/594.`, true)
```

Comment:
I think this never gets put anywhere since the `generate-default-config` package script is broken (and always has been IIRC), but the same comments are elsewhere, so it's fine.